src/components/auth/login/LoginForm.js

```diff
@@ -19,7 +19,7 @@ export const LoginForm = (props) => {
             successSwalTitle: "Successfully logged in!",
             successFunction: (token) => {
                 reset()
-                alert([...token])
+                console.log(token)
```

Comment:
Potrzebujemy logować token?

Reply:
na razie tak, bo w sumie to nie wiem dalej, co tam się dzieje do końca XDDD

Reply:
Tak dla sami wiemy kogo, żeby się mógł pośmiać ❤️